src/integration-test/java/developerTC/AgentService.java

```diff
@@ -155,32 +129,10 @@ public static RemoteAgent getRemoteAgent() {
         return surfer;
 
     }
-    public static boolean isServerReachable(String uri) {
```

Comment:
Can you test the Agent Status functionality here (DEP 11)?

Reply:
Agent Status is not yet implemented, will do tomorrow